sgx/test/src/kmyth_sgx_test.c

```diff
@@ -60,13 +85,105 @@ void test_enclave_seal_unseal(void)
                   in_size);
   CU_ASSERT(sgx_ret == 0);
   CU_ASSERT(memcmp(in_data, out_data_decrypted, in_size) == 0);
-
   free(in_data);
   free(out_data_decrypted);
   free(out_data);
   return;
 }
 
+void test_unseal_and_export(void)
+{
+  uint8_t *plain_data = NULL;
+  uint8_t **cipher_data = NULL;
+  uint64_t *handles = NULL;
+  size_t num_ciphertexts = 5;
+
+  uint8_t *cipher_data_decrypted = NULL;
+
+  size_t plain_size = sizeof(size_t);
+  size_t cipher_size = 0;
+
+  uint16_t key_policy = SGX_KEYPOLICY_MRSIGNER;
+  sgx_attributes_t attribute_mask;
+
+  attribute_mask.flags = 0;
+  attribute_mask.xfrm = 0;
+
+  int sgx_ret_int;
+  size_t sgx_ret_size_t;
+
+  enc_get_sealed_size(eid, &sgx_ret_int, plain_size,
+                      (uint32_t *) & cipher_size);
+  CU_ASSERT(sgx_ret_int == 0);
+
+  plain_data = (uint8_t *) calloc(plain_size, 1);
+
+  cipher_data = (uint8_t **) malloc(3 * sizeof(uint8_t *));
```

Comment:
Cutting out some of the other code...

    size_t num_ciphertexts = 5;
    cipher_data = (uint8_t **) malloc(3 * sizeof(uint8_t *));
    for (size_t i = 0; i < num_ciphertexts; i++)
    {
      cipher_data[i] = (uint8_t *) malloc(cipher_size);

It looks like you're allocating space for 3 pointers then using 5.